java.base/src/main/java/java/lang/Thread$_patch.java

```diff
@@ -0,0 +1,254 @@
+/*
+ * This code is based on OpenJDK source file(s) which contain the following copyright notice:
+ *
+ * ------
+ * Copyright (c) 1994, 2019, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ * ------
+ *
+ * This file may contain additional modifications which are Copyright (c) Red Hat and other
+ * contributors.
+ */
+
+package java.lang;
+
+import static org.qbicc.runtime.CNative.*;
+import static org.qbicc.runtime.linux.Futex.*;
+import static org.qbicc.runtime.posix.PThread.*;
+import static org.qbicc.runtime.posix.Time.*;
+import static org.qbicc.runtime.stdc.Stdint.*;
+import static org.qbicc.runtime.stdc.Time.*;
+
+import java.security.AccessControlContext;
+
+import org.qbicc.runtime.Build;
+import org.qbicc.runtime.patcher.Add;
+import org.qbicc.runtime.patcher.PatchClass;
+import org.qbicc.runtime.patcher.Replace;
+
+/**
+ * Patches for {@code java.lang.Thread}.
+ */
+@PatchClass(Thread.class)
+public class Thread$_patch {
+    // alias fields
+    String name;
+    ThreadGroup group;
+    boolean daemon;
+    int priority;
+    ClassLoader contextClassLoader;
+    AccessControlContext inheritedAccessControlContext;
+    Runnable target;
+    ThreadLocal.ThreadLocalMap inheritableThreadLocals;
+    long stackSize;
+    long tid;
+    int threadStatus;
+
+    // alias methods
+    native void setPriority(final int priority);
+    static native long nextThreadID();
+
+    // used only by Linux
```

Comment:
I think the comment is backwards?  Used only when not Linux?

Reply:
Fixed.